### PR TITLE
Adds support needed for mapping file

### PR DESCRIPTION
**Pull request by scanon**
This addresses issue #132

### CODE REVIEW

File: tests/workflows_test.yaml [hunk 1]
```diff
@@ -297,11 +298,12 @@ Workflows:
...
-      #   description: Contig mappings file for {id}
-      #   name: Contig mappings between contigs and scaffolds
-      #   suffix: _contig_names_mapping.tsv
+      - output: contig_mapping
+        data_object_type: Contig Mapping File
+        description: Contig mappings file for {id}
+        name: Contig mappings between contigs and scaffolds
+        suffix: _contig_names_mapping.tsv
+        optional: True
       - output: imgap_version
```
Comment (aclum) [inline tests/workflows_test.yaml:307]: Also add the assembly file from *_contigs.fna file with a data_object_type of 'Assembly Contigs' as an output file generated by [MetagenomeAnnotationActivity](https://microbiomedata.github.io/nmdc-schema/MetagenomeAnnotationActivity/)

**Review by aclum — CHANGES_REQUESTED** [review]
MAGs should use version v1.3.1
Annotation should use version v1.1.0

File: tests/workflows_test.yaml (hunk 1 above)
Comment (scanon) [reply tests/workflows_test.yaml:307]: I'm going to do a second PR that will update the config.  So this will just be focused on the code changes.

**Comment by scanon** [comment]
I'm going to merge this PR and I will do the other updates to the config in a separate PR

**Pull request merged**